lib/file-source/Cargo.toml

```diff
@@ -14,6 +14,8 @@ scan_fmt = "0.2.3"
 tracing = "0.1.2"
 indexmap = {version = "1.0.2", features = ["serde-1"]}
 flate2 = "1.0.6"
+winapi = { version = "0.3.8", features = ["winioctl"] }
```

Comment:
I think we pullin `winapi 0.2` via `mio 0.6` to avoid a duplicate here for now, is it possible to use `winapi 0.2`? I don't think this should hurt compile times either since we already need to compile it.

Reply:
Fantastic, great catch. :) I'll verify this and make the change.

Reply:
Seems winapi 2.x doesn't have some variables like ][`FSCTL_GET_REPARSE_POINT`](https://docs.rs/winapi/0.2.8/winapi/?search=FSCTL_GET_REPARSE_POINT). :(

Reply:
thats fine, its windows anyways :)